docker/server/entrypoint.sh

```diff
@@ -38,6 +38,7 @@ FORMAT_SCHEMA_PATH="$(clickhouse extract-from-config --config-file "$CLICKHOUSE_
 
 # There could be many disks declared in config
 readarray -t DISKS_PATHS < <(clickhouse extract-from-config --config-file "$CLICKHOUSE_CONFIG" --key='storage_configuration.disks.*.path' || true)
+readarray -t DISKS_METADATA_PATHS < <(clickhouse extract-from-config --config-file "$CLICKHOUSE_CONFIG" --key='storage_configuration.disks.*.metadata_path' || true)
```

Comment:
why? if there is such code:
https://github.com/ClickHouse/ClickHouse/blob/583c530d0f2914804fd5681d041aecc97076eae0/src/Disks/ObjectStorages/DiskObjectStorageCommon.cpp#L26-L27